include/rangers-api/Application/Game/GameMode.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+namespace app::game {
+    class GameMode : public hh::fnd::Messenger {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.